docs/source/guidelines.rst

```diff
@@ -1,48 +1,200 @@
 .. _Guidelines:
 
-Guidelines for Developers
-*************************
+Guidelines for Gammasim-tools Developers
```

Comment:
We never wrote `Gammasim-tools` with `Gamma` capitalised. Everywhere else it is `gammasim-tools`. If you don't mind, I would modify it here to be `gammasim-tools`.